docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md

```diff
@@ -279,3 +279,129 @@ by the `OPERATING_SYSTEM` and all basic keys work according to their labels.
 1. All standard keyboard keys generate correct characters
    or actions when pressed.
 1. Key combinations are detected correctly.
+
```

Comment:
@Pokisiekk How about test-matrixes for various variants? We are going to add these test cases for all platforms?

Reply:
Where the value of `Supported test cases` tab is `All` in the line with the test module, the tests are automatically added to the test-matrix. Where a manual change was required, I did it - TalosII and Protectli fw6